README.md

```diff
@@ -34,7 +41,28 @@ This code requires Python 3.6 or a higher version. If you haven’t already, dow
 ```
 python icone_translator.py -i inputfile.xml -o outputfile.geojson
 ```
+Example usage:
+```
+python icone_translator.py -i '../sample files/icone data/incidents_extended.xml' 
+```
+
+### Testing
+
+#### Step 1: Run the unit test for translator script (from root directory)
+
+```
+coverage run -m pytest 'test/' -v 
```

Comment:
Running this doesn't work out of the box. What is "coverage"? Is this an additional tool required to perform the testing? If so we need to mention that here.

Reply:
i changed it ..can you please take a look and let me know if thats ok or not.Thanks

Reply:
There is a pip install line in the readme, `pip install -r requirements.txt`. If you run that line (from root) and then try the coverage line, does it work?

Reply:
@jacob6838, I don't see a reference to "coverage" in the requirements.txt. But I did run `pip install -r requirements.txt` and the coverage call doesn't work. I'm running Windows 10 if that matters, maybe there's an extra step to installing coverage on Windows?

Reply:
Note I can run `python -m pytest .\test\ -v` successfully, though it does add a `__pycache__` directory so we'll need to update the .gitigore file for the standard Python set.

Reply:
It should be installed with `pytest-cov`, hmm...

Reply:
My mistake, coverage is something else. You are right, the command should be `python -m pytest .\test\`. We just updated this in the main readme